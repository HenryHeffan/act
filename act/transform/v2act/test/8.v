module blk1(a,b,out);
  output [2:0] out;
  input [1:0] a;
  input b;
  
 assign {out[2], out[1], out[0]} = {a[1:0],b};
endmodule
