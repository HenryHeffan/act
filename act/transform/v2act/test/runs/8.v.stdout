/* -- declarations -- */
defproc blk1 (bool? a[2]; bool? b; bool! out[3]);

defproc blk1 (bool? a[2]; bool? b; bool! out[3])
{
   /*--- types ---*/
   /*--- connections ---*/
   out[2]=a[1];
   out[1]=a[0];
   out[0]=b;
}

